File: Assignment36_3.java
import java.util.Scanner;

///////////////////////////////////////////////////////////////////////////////
// Function Name : strtogglex
// Description   : Toggle the case of each character of string
// Input         : String
// Output        : Modified String
// Author        : Raj Samir Jadhav
// Date          : 30/11/2025
///////////////////////////////////////////////////////////////////////////////
class Pattern
{
    public String strtogglex(String str)
    {
        String result = "";

        for(int i = 0; i < str.length(); i++)
        {
            char ch = str.charAt(i);

            if(ch >= 'a' && ch <= 'z')           // If small convert to capital
            {
                ch = (char)(ch - 32);
            }
            else if(ch >= 'A' && ch <= 'Z')       // If capital convert to small
            {
                ch = (char)(ch + 32);
            }

            result = result + ch;
        }
        return result;
    }
}

class Assignment36_3
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter string : ");
        String input = sobj.nextLine();

        Pattern pobj = new Pattern();
        String output = pobj.strtogglex(input);

        System.out.println("Modified string is : " + output);

        sobj.close();
    }
}

File: Assignment36_3.c
#include <stdio.h>

///////////////////////////////////////////////////////////////////////////////
// Function Name : strtogglex
// Description   : Toggle the case of each character of string
// Input         : String
// Output        : Modified String
// Author        : Raj Samir Jadhav
// Date          : 30/11/2025
///////////////////////////////////////////////////////////////////////////////

void strtogglex(char *str)
{
    while(*str != '\0')                              // Traverse entire string
    {
        if((*str >= 'A') && (*str <= 'Z'))           // If capital letter → convert to small
        {
            *str = *str + 32;
        }
        else if((*str >= 'a') && (*str <= 'z'))       // If small letter → convert to capital
        {
            *str = *str - 32;
        }
        str++;
    }
}

int main()
{
    char arr[30];

    printf("Enter string : ");
    scanf("%[^'\n']s",arr);

    strtogglex(arr);                                     // Function call

    printf("Modified string is : %s\n",arr);

    return 0;
}
